ext/net/03_quic.js

```diff
@@ -309,6 +315,43 @@ class QuicConn {
   close({ closeCode = 0, reason = "" } = { __proto__: null }) {
     op_quic_connection_close(this.#resource, closeCode, reason);
   }
+
+  static {
+    webtransportConnect = async function webtransportConnect(conn, url) {
+      const {
+        0: connectTxRid,
+        1: connectRxRid,
```

Comment:
Not sure if I fully grasp this - why are these two functions created in a static block of a class?